homeassistant/components/asuswrt/__init__.py

```diff
@@ -125,6 +125,14 @@ async def async_setup_entry(hass: HomeAssistantType, entry: ConfigEntry):
 
     router.async_on_close(entry.add_update_listener(update_listener))
 
+    # Device ID was changed. Link up old device to new one.
+    dev_reg = device_registry.async_get(hass)
+    legacy_entry = dev_reg.async_get_device((DOMAIN, "AsusWRT"))
```

Comment:
I think this should be `legacy_entry = dev_reg.async_get_device({(DOMAIN, "AsusWRT")})`

Reply:
```suggestion
    legacy_entry = dev_reg.async_get_device({(DOMAIN, "AsusWRT")})
```